__tests__/get-release-names.test.ts

```diff
@@ -0,0 +1,120 @@
+import {getReleaseNames} from '../src/get-release-names'
+import github from '@actions/github'
+
+jest.mock('@actions/github', () => ({
+  __esModule: true,
+  default: {
+    getOctokit: jest.fn()
+  }
+}))
+
+type Octokit = ReturnType<typeof github['getOctokit']>
+type ListReleasesFn = Octokit['rest']['repos']['listReleases']
+type ListReleasesResult = ReturnType<ListReleasesFn> extends Promise<infer T>
+  ? T
+  : never
+type Release = ListReleasesResult['data'] extends (infer T)[] ? T : never
+
+function setupReleaseList(tags?: string[]): jest.Mock {
+  const listReleases = jest.fn<
+    Promise<ListReleasesResult>,
+    Parameters<ListReleasesFn>
+  >()
+  const octokit = {
+    rest: {
+      repos: {
+        listReleases
+      }
+    }
+  }
+  ;(github.getOctokit as jest.Mock).mockReturnValue(octokit)
```

Comment:
that `;(` syntax is new to me, I see it in a few places in the PR where you're mocking stuff.

Reply:
It's just a workaround for a weird syntactical thing when you're not using semicolons (again, the project came with Prettier config that's different than what we normally use).
Because the last line ended in an expression, without the semicolon here, JS would interpret these parenthesis as a function call (which would fail, because said expression is an object literal) instead of an expression itself.

There's a reason we typically use semicolons. Look, even the syntax is crying. `;(`